x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/entsearch/engine/Engine.java

```diff
@@ -41,6 +41,7 @@
 public class Engine implements Writeable, ToXContentObject {
     private final String name;
     private final String[] indices;
+    private final long updatedAt;
     private final String analyticsCollectionName;
```

Comment:
I wonder if we should also add the aliasName? We don't need to do it in this PR though.

Reply:
For now the alias is always using the same prefix - I don't think we need that unless users are able to change it.

Reply:
Great catch! I was just talking to @TattdCodeMonkey about this earlier in the week. We do need to add the alias name for the front-end (so it's not relying on rules). I was thinking of doing it in a separate PR but if we're changing the serialization we may as well do it now.